### PR TITLE
Disable compression to query Live Views using `clickhouse-js` adapter

### PR DESCRIPTION
**Pull request by Wain-PC**
As a follow-up from [#157](https://github.com/ClickHouse/clickhouse-js/issues/157), a found issue should be noted in the docs.

### CODE REVIEW

**Comment by slvrtrn** [comment]
@Wain-PC 
Looks good! Could you sign the CLA?

File: docs/en/integrations/language-clients/nodejs.md [hunk 1]
```diff
@@ -734,6 +734,7 @@ TLS in the repository.
 - There are some [Decimal* and Date\* / DateTime\* data types caveats](#date--datetime-types-caveats).
 - [Nested](/docs/en/sql-reference/data-types/nested-data-structures/index.md) data type is currently not officially
   supported.
+- [Response compression](#compression) must be [disabled](https://github.com/ClickHouse/clickhouse-js/issues/157#issuecomment-1546005694) when using [Live Views](/docs/en/sql-reference/statements/create/view#live-view-experimental)
```
Comment (DanRoscigno) [inline docs/en/integrations/language-clients/nodejs.md:737]:
```suggestion
- [Response compression](#compression) must be [disabled](https://github.com/ClickHouse/clickhouse-js/issues/157#issuecomment-1546005694) when using [Live Views](/docs/en/sql-reference/statements/create/view.md/#live-view-experimental)
```

Comment (DanRoscigno) [reply docs/en/integrations/language-clients/nodejs.md:737]: links to the .md files are verified by Docusaurus

**Review by DanRoscigno — APPROVED** [review]

**Comment by robot-clickhouse-ci-2** [comment]
This is an automated comment for commit d3b1fd318ae4dcc6b9d044bab2b8a6f78ced70dc with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1151/d3b1fd318ae4dcc6b9d044bab2b8a6f78ced70dc/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🔴 failure</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

**Pull request merged**